src/main/archetype/it.tests/pom.xml

```diff
@@ -41,9 +41,9 @@
             <version>4.13.1</version>
         </dependency>
         <dependency>
-            <groupId>net.sourceforge.htmlunit</groupId>
```

Comment:
Why are these changes part of this PR ?

Reply:
Its fixed now. Seems like it was a part of #1070 which was being shown here for some weird reason.